client/client-lib/src/ln/db.rs

```diff
@@ -12,6 +15,7 @@ const DB_PREFIX_OUTGOING_PAYMENT_CLAIM: u8 = 0x24;
 const DB_PREFIX_OUTGOING_CONTRACT_ACCOUNT: u8 = 0x25;
 const DB_PREFIX_CONFIRMED_INVOICE: u8 = 0x26;
 const DB_PREFIX_LIGHTNING_GATEWAY: u8 = 0x28;
+const DB_PREFIX_PAYMENTS: u8 = 0x29;
```

Comment:
requires an update of the [DB docs](https://github.com/fedimint/fedimint/blob/master/docs/database.md)